src/poketch_digital_clock/poketch_digital_clock_manager.c

```diff
@@ -0,0 +1,239 @@
+#include <nitro.h>
+#include <nitro/sinit.h>
+#include <string.h>
+
+#include "overlay025/poketch_button.h"
+#include "overlay025/poketch_system.h"
+#include "poketch_digital_clock/poketch_digital_clock_graphics.h"
+
+#include "bg_window.h"
+#include "heap.h"
+#include "rtc.h"
+#include "sys_task.h"
+#include "sys_task_manager.h"
+#include "touch_screen.h"
+
+#define BACKLIGHT_BUTTON_MIN_Y 0x10
+#define BACKLIGHT_BUTTON_MAX_Y 0xAF
+#define BACKLIGHT_BUTTON_MIN_X 0x10
+#define BACKLIGHT_BUTTON_MAX_X 0xCF
+
+#define APP_TASK_LOAD_APP   0
+#define APP_TASK_UPDATE_APP 1
+#define APP_TASK_UNLOAD_APP 2
+
+typedef struct {
+    u8 appTasktaskID;
+    u8 taskStage;
```

Comment:
```suggestion
    u8 state;
```
To be consistent across the repo